src/commands/createFunction/FunctionCreateStepBase.ts

```diff
@@ -53,6 +56,15 @@ export abstract class FunctionCreateStepBase<T extends IFunctionWizardContext> e
         await verifyExtensionBundle(context, template);
 
         const cachedFunc: ICachedFunction = { projectPath: context.projectPath, newFilePath, isHttpTrigger: template.isHttpTrigger };
+        if (context.functionTemplate?.defaultFunctionName === 'ServiceBusQueueTrigger' || context.functionTemplate?.defaultFunctionName === 'BlobTrigger' || context.functionTemplate?.defaultFunctionName === 'QueueTrigger') {
```

Comment:
You shouldn't use properties like `defaultFunctionName`, `name`, or `label` when you can help it. There's always a chance that these could change, and sometimes localization can screw these conditionals up.

I think the preferred method would be to check the `id` property instead.

Also, rather than have a conditional like this, I think it'd be better to change the `IFunctionTemplate` to have a property that is `isDynamicConcurrent` (or something like that) which would have this check. It would be a little bit more tidy 😊